src/stylesheets/livefyre_widget_overrides.scss

```diff
@@ -469,9 +469,9 @@
 		}
 
 		.fyre-comment-flag-mobile-btn {
-			
+
```

Comment:
I don't think it is a new line, just Github getting confused. Try viewing without whitespace changes -- https://github.com/Financial-Times/o-comments/pull/11/files?w=1